### ISSUE TITLE
canDebug=false on limelight_edgefunction resource is not passed

### ISSUE THREAD

**Issue by onoel** [llnw/terraform-provider-limelight#11]
I have an EdgeFunctions function configuration which is defined as 

```
{
  "canDebug": true,
  "description": "Example",
  "environmentVariables": [],
  "handler": "example.handler",
  "memory": 1024,
  "name": "example",
  "reservedConcurrency": 0,
  "revisionId": 11,
  "runtime": "python3",
  "sha256": "e48b07c33e99ca32eacec0c05390b19bdb5978153fe405ee589b50cec476075b",
  "timeout": 5000,
  "version": "$LATEST"
}
```

When I changing the configuration from `canDebug=true` to `canDebug=false`, the `canDebug` value is not updated during the request to the API `updateFunctionConfiguration`. 

Here is the "diff" output of the `terraform apply` command 

```
  # limelight_edgefunction.example will be updated in-place
  ~ resource "limelight_edgefunction" "example_function" {
      ~ can_debug            = true -> false
        description          = "Example"
        function_archive     = "function.zip"
        function_sha256      = "c2d19ad352ea002461120c3d3ab3a2a34859e193f03f81a6fd77517f439a79382"
        handler              = "example.handler"
        id                   = "myShortname:example"
        memory               = 512
        name                 = "example"
        reserved_concurrency = 0
        revision_id          = 8
        runtime              = "python3"
        shortname            = "myShortname"
        timeout              = 10000
    }
```

**Comment by colinbarr** [comment]
Good spot @onoel. This was happening because the `canDebug` attribute had been tagged with the `omitempty` flag. Because the `false` value is considered "empty", `canDebug` was not sent in the request body to the EdgeFunctions API. As a result, the API retained the previous setting which effectively meant you could enable the debug mode, but never disable.

**Comment by colinbarr** [comment]
Hi @onoel, this has been fixed in v1.0.3 of the provider.
